ab3d2_source/hires.s

```diff
@@ -6740,9 +6740,18 @@ doneclip:
 				lea		(a4,d1*2),a4			; go to top linetab
 ; move.l #dists,a2
 				move.w	View2FloorDist,d0
-				;muls	#64,d0			; FIXME: why muls here? Is this addressing the floor tile row?
-				ext.l	d0
-				lsl.l	#6,d0
+***************************************************************
+;could this be screen width * 0.33333 ?
+				tst.b	Vid_FullScreen_b
+				beq.s .smallscreen
+				muls	#107,d0
```

Comment:
We should indicate what we are approximating with this multiplier.

Reply:
not sure how to put this into words that will make sense.